packages/flutter_goldens/test/flutter_goldens_test.dart

```diff
@@ -34,15 +34,13 @@ void main() {
   });
 
   group('GoldensClient', () {
-    GoldensClient goldens;
+    GoldensRepositoryClient goldens;
 
-    setUp(() {
-      goldens = GoldensClient(
-        fs: fs,
-        platform: platform,
-        process: process,
-      );
-    });
+    setUp(() => goldens = GoldensRepositoryClient.test(
```

Comment:
nit: us {} instead of => since the whole thing doesn't fit on one line. (see style guide).